migrations/2025-06-24-120137_disable-post-notifications/up.sql

```diff
@@ -1,9 +1,9 @@
-CREATE TYPE post_notifications_mode_enum AS enum (
+CREATE TYPE notifications_mode_enum AS enum (
     'RepliesAndMentions',
-    'AllComments',
+    'All',
     'Mute'
 );
 
 ALTER TABLE post_actions
-    ADD COLUMN notifications post_notifications_mode_enum;
+    ADD COLUMN notifications notifications_mode_enum;
 
```

Comment:
Had to change this to use the same enum for post subscriptions and community subscriptions. Maybe its better to revert the previous PR and make all the changes here in a single migration.